device/common/include/traccc/clusterization/device/impl/aggregate_cluster.ipp

```diff
@@ -107,6 +107,9 @@ inline void aggregate_cluster(
     out.local = mean;
     out.variance = var;
     out.module_link = module_link;
+    // The following two will need to be filled properly "soon".
+    out.meas_dim = 2u;
+    out.surface_link = detray::geometry::barcode{};
```

Comment:
Can you try this?
```c++
out.surface_link = detray::geometry::barcode{this_module.module}
```

Reply:
Thanks for the tip!

I also added explicit initialization for the `surface_link` value in the host code, to keep the two in sync. :wink: